host.md

```diff
@@ -82,8 +84,10 @@ If internet connectivity were to fail, the meeting would need to be suspended.
 - There must be at least one large main display visible to everyone for slides, etc.
   - Ideally there would be two large displays: one for slides, one for showing video for remote participants.
 - There must be at least 8 individual high-quality low-latency microphones and they must have a mute function.
+  - Ideally there would be a ratio of one microphone for every three attendees.
```

Comment:
"at least"? Also might want to clarify that this is for table or handheld mics, not room mics.